spring-boot/proxy/src/main/java/eu/coatrack/proxy/filters/pre/ApiKeyAuthFilter.java

```diff
@@ -43,25 +45,33 @@
  *
  * @author gr-hovest
  */
-public class ApiKeyAuthFilter extends GenericFilterBean {
+public class ApiKeyAuthFilter extends OncePerRequestFilter {
```

Comment:
This is in general a good idea: we should probably update all our classes extending `GenericFilterBean` to extend `OncePerRequestFilter` instead, because imho none of them needs to run more than once per request.

Reply:
All classes containing `GenericFilterBean` changed to `OncePerRequestFilter`, so this is done.